packages/react/src/components/Pivot/Pivot.types.ts

```diff
@@ -88,6 +89,12 @@ export interface IPivotProps extends React.HTMLAttributes<HTMLDivElement>, React
    * Useful if you're rendering content outside and need to connect aria-labelledby.
    */
   getTabId?: (itemKey: string, index: number) => string;
+
+  /**
+   * Use this prop if you would like to override default behaviour of horizontal tabbing.
+   * @default FocusZoneDirection.horizontal
+   */
+  focusZoneDirection?: FocusZoneDirection.horizontal;
```

Comment:
I think it would be better if we introduced `focusZoneProps?: IFocusZoneProps` so that we could pass more than just the individual `FocusZone` direction.